Chinese/Kong-Shi-foo/Kong-Shi-foo.md

```diff
@@ -1,3 +1,12 @@
+## 斗战胜佛转世做泡面 康师傅借此统一华夏天
+
 孙悟空转世成了康师傅，于是开始做方便面，借此一统天下。
 
-作者按：这都是什么乱七八糟的故事啊？怎么还扯到大话西游上来了？汗。咱弄一个冒险小说不行吗？
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.